securedrop-log-saver

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import redis
```

Comment:
will this require a pip install?

Reply:
Yeah, which of course will need to be done in the template.